tailwind.config.js

```diff
@@ -226,6 +228,14 @@ module.exports = {
                         transform: 'translate3d(4px, 0, 0)',
                     },
                 },
+                slideDown: {
+                    from: { height: '0', opacity: 0 },
+                    to: { height: 'var(--radix-accordion-content-height)', opacity: 1 },
+                },
+                slideUp: {
+                    from: { height: 'var(--radix-accordion-content-height)', opacity: 1 },
+                    to: { height: '0', opacity: 0 },
+                },
```

Comment:
Given that these animations are specific to the accordion component, I would move them to a css files inside the `/accordion` folder (`/accordion/index.css`) and import it from the `/components/index.css` file, something like:
```
@keyframes accordionExpand {
  from: ...
  to: ..
}

@keyframes accordionCollapse {
  from: ...
  to: ..
}
```
And use them inline as `animate-[accordionCollapse_0.3s_cubic-bezier(0.87, 0, 0.13, 1)_forwards]`